tests/python-pytest/onnx/import/test_cases.py

```diff
@@ -98,6 +98,7 @@
     'test_MaxPool',
     'test_PReLU',
     'test_ReLU',
+    'test_selu_default',
```

Comment:
why is `test_selu` being excluded?

Reply:
alpha and lambda are fixed based on the paper arxiv:1706.02515 (Eqn 14). for self-normalizing property of SNNs, this fixed alpha and lambda value is required.
the tests in ONNX for selu were added as "testing math operators" and different alpha/lambda values were provided.

Reply:
I see, so alpha and lambda cant be passed as parameters to `LeakyRelu` which has `selu` as an activation type. That might be a bug.

Reply:
update based on offline discussions - MXNet's implementation fixed these values based on the recommendation in the paper, so alpha and lambda will continue to be the default values and not configurable parameters to LeakyRelu. following this, only test_selu_default will be added as a test for ONNX import/export.